utils/display/apps.go

```diff
@@ -35,12 +29,15 @@ func AppsTable(apps []*meroxa.Application, hideHeaders bool) string {
 			{Align: simpletable.AlignLeft, Text: string(app.Status.State)},
 		}
 
-		env, err := app.Environment.GetNameOrUUID()
-
-		if err == nil && itShouldEnvInfo(app.Environment) {
-			withEnvironment = true
-			r = append(r, &simpletable.Cell{Align: simpletable.AlignLeft, Text: env})
+		if app.Environment != nil {
+			env, err := app.Environment.GetNameOrUUID()
+			if err == nil {
+				r = append(r, &simpletable.Cell{Align: simpletable.AlignLeft, Text: env})
+			}
```

Comment:
The error case isn't getting handled here. Instead I recommend being explicit about whether you want to include the name or the uuid here. 

Here's an example for resources https://github.com/meroxa/cli/blob/2a0ff49ec84721823307244176272a18b0bd5437/utils/display/resources.go#L53-L65